Run TCAV.ipynb

```diff
@@ -271,7 +268,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "utils.print_results(results)"
+    "utils.print_results(results, output_plot=True, num_random_exp=10)"
```

Comment:
update jupyter notebook to not set output_plot=True here and instead use utils_plot to show plot